W-19128919 agent fabric exchange GA - jh (#319)
mulesoft/docs-exchange: modules/ROOT/pages/index.adoc

:imagesdir: ../assets/images/
:page-aliases: general::getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc about-exchange1.adoc, exchange1.adoc, exchange1-faq.adoc

> Anypoint Exchange is a curated catalog of reusable assets. Agents, APIs, API groups, API spec fragments, custom assets, examples, GraphQL APIs, large learning models (LLMS), Model Control Protocol (MCP) servers, integration assets such as connectors, policies, RPA assets, rulesets, and templates are some of the types that are supported in Exchange. See xref:asset-types.adoc[Asset Types] for more information about each type of asset. You can catalog (publish), share, discover, learn about, and reuse assets within your organization of developers to facilitate collaboration, boost productivity, and promote standards.

Reviewer: ```suggestion
Anypoint Exchange is a curated catalog of reusable assets. Agents, APIs, API groups, API spec fragments, custom assets, examples, GraphQL APIs, large language models (LLMs), Model Control Protocol (MCP) servers, integration assets such as connectors, policies, RPA assets, rulesets, and templates are some of the types that are supported in Exchange. See xref:asset-types.adoc[Asset Types] for more information about each type of asset. You can catalog (publish), share, discover, learn about, and reuse assets within your organization of developers to facilitate collaboration, boost productivity, and promote standards.
```

> Anypoint Exchange is a curated catalog of reusable assets. Agents, APIs, API groups, API spec fragments, custom assets, examples, GraphQL APIs, Large Learning Modeles (LLMS), Media Control Protocol (MCP) servers, integration assets such as connectors, policies, RPA assets, rulesets, and templates are some of the types that are supported in Exchange. See xref:asset-types.adoc[Asset Types] for more information about each type of asset. You can catalog (publish), share, discover, learn about, and reuse assets within your organization of developers to facilitate collaboration, boost productivity, and promote standards.

Reviewer 2: ```suggestion
Anypoint Exchange is a curated catalog of reusable assets. Agents, APIs, API groups, API spec fragments, custom assets, examples, GraphQL APIs, Large Learning Modeles (LLMS), Model Context Protocol (MCP) servers, integration assets such as connectors, policies, RPA assets, rulesets, and templates are some of the types that are supported in Exchange. See xref:asset-types.adoc[Asset Types] for more information about each type of asset. You can catalog (publish), share, discover, learn about, and reuse assets within your organization of developers to facilitate collaboration, boost productivity, and promote standards.
```

image:exchange-landing-page.png["Exchange catalog showing assets provided by MuleSoft"]

> Use Exchange to discover, share, incorporate assets and resources into your applications, and view asset usage and engagement metrics. You can create developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

Reviewer: ```suggestion
Use Exchange to discover, share, and incorporate assets and resources into your applications, and view asset usage and engagement metrics. You can create developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
```

> Exchange assets are accessible from many places, such as Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), or Runtime Manager (applications). Agents, APIs, and MCP servers can be published from Anypoint Code Builder to Exchange.

Reviewer 2: I wonder if we should say something like: 
You can publish assets from agent network projects such as agents, brokers, and MCP servers, as well as APIs from...

== MuleSoft Assets
Assets in the *Provided by MuleSoft* organization are created by MuleSoft and its partners and certified by MuleSoft. These assets are publicly available to all customers. However, assets that are created by an organization other than *Provided by MuleSoft* and published in that organization's instance of Exchange are private and not visible outside that organization. Customers are responsible for any private assets they publish within their Exchange instance.

== Agent Fabric Assets

> Agent Fabric assets include agentic assets and tools to provide support and automate tasks. You can find agent, Large Language Model (LLM), and MCP server assets in the *Agents & Tools (New)* tab. 

Reviewer 2: ```suggestion
Agent Fabric assets include agentic assets and tools to provide support and automate tasks. You can find agent, large language model (LLM), and MCP server assets in the *Agents & Tools (New)* tab. 
```

An Exchange administrator sets the groups that this menu displays for each user.

> If you cannot find an asset in the current business group scopes, search in other scopes. In the navigation menu, click *All assets* (assets provided by MuleSoft and your own root organization), *Provided by MuleSoft*, or a business group scope.

Reviewer 2: ```suggestion
If you can't find an asset in the current business group scopes, search in other scopes. In the navigation menu, click *All assets* (assets provided by MuleSoft and your own root organization), *Provided by MuleSoft*, or a business group scope.
```
